Gemfile

```diff
@@ -20,10 +20,10 @@ gem 'bootsnap', '>= 1.4.2', require: false
 
 gem 'action_policy'
 gem 'assembly-objectfile', '~> 1.9'
-gem 'cocina-models', '~> 0.65.0'
+gem 'cocina-models', '~> 0.66.0'
 gem 'committee'
 gem 'config', '~> 2.0'
-gem 'dor-services-client', '~> 7.8'
+gem 'dor-services-client', '~> 7.12'
```

Comment:
This change isn't *really* necessary, because '~> 7.8' matches with 7.12.0, but NBD also.